### PR TITLE
[INLONG-6972][Dashboard]  All "consumption" is changed to "subscription" for data consumption

### LINKED ISSUE #6972: [Improve][Dashboard] Change all "consumption" to "subscription" for data consumption
### Description

image

### InLong Component

InLong Dashboard

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by kinfuy**
### Prepare a Pull Request

- Fixes #6972

### CODE REVIEW

File: inlong-dashboard/src/locales/cn.json [hunk 1]
```diff
@@ -363,18 +363,18 @@
...
-  "meta.Consume.ConsumerGroupName": "消费组名称",
+  "meta.Consume.ConsumerGroupName": "订阅组名称",
   "meta.Consume.ConsumerGroupNameRules": "只能包含小写字母、数字、中划线、下划线",
   "meta.Consume.TopicName": "Topic名称",
   "meta.Consume.MQType": "MQ类型",
   "meta.Consume.TargetInlongGroupID": "所属的数据流组ID",
   "meta.Consume.TargetInlongStreamID": "所属的数据流ID",
   "meta.Consume.MQAddress": "消息队列地址",
-  "meta.Consume.FilterEnabled": "是否过滤消费",
+  "meta.Consume.FilterEnabled": "是否过滤订阅",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:373]: Keep the "是否过滤消费" here

File: inlong-dashboard/src/locales/cn.json [hunk 2]
```diff
@@ -535,30 +535,30 @@
...
   "pages.ConsumeDashboard.config.Reject": "已驳回",
   "pages.ConsumeDashboard.config.OperatingStatus": "运行状态",
-  "pages.ConsumeDashboard.config.Total": "消费总数",
+  "pages.ConsumeDashboard.config.Total": "订阅总数",
   "pages.ConsumeDashboard.config.Pending": "待审批",
-  "pages.ConsumeDashboard.config.ConsumerGroup": "消费组",
+  "pages.ConsumeDashboard.config.ConsumerGroup": "订阅组",
   "pages.ConsumeDashboard.config.ConsumeInlongGroupId": "数据流组ID",
-  "pages.ConsumeDashboard.config.RecentConsumeTime": "最近消费时间",
+  "pages.ConsumeDashboard.config.RecentConsumeTime": "最近订阅时间",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:552]: keep the "最近消费时间"

File: inlong-dashboard/src/locales/cn.json [hunk 3]
```diff
@@ -363,18 +363,18 @@
   "meta.Stream.ExecuteWorkflow": "执行工作流",
   "meta.Stream.ExecuteConfirm": "确认执行工作流吗?",
   "meta.Stream.ExecuteSuccess": "执行成功",
-  "meta.Consume.ConsumerGroupName": "消费组名称",
+  "meta.Consume.ConsumerGroupName": "订阅组名称",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:366]: keep the "消费组名称"

File: inlong-dashboard/src/locales/cn.json [hunk 4]
```diff
@@ -535,30 +535,30 @@
...
+  "pages.ConsumeCreate.NewConsume": "新建订阅",
   "pages.ConsumeDashboard.config.BeAllocated": "待分配",
   "pages.ConsumeDashboard.config.ApplicationStatus": "申请状态",
   "pages.ConsumeDashboard.config.Reject": "已驳回",
   "pages.ConsumeDashboard.config.OperatingStatus": "运行状态",
-  "pages.ConsumeDashboard.config.Total": "消费总数",
+  "pages.ConsumeDashboard.config.Total": "订阅总数",
   "pages.ConsumeDashboard.config.Pending": "待审批",
-  "pages.ConsumeDashboard.config.ConsumerGroup": "消费组",
+  "pages.ConsumeDashboard.config.ConsumerGroup": "订阅组",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/cn.json:550]: keep the "消费组"

**Comment by dockerzhang** [comment]
Can we change all English “Consumes” to ”Subscribe“, and “Consumer” to "Subscriber" at the same time?

like 
"configs.menus.Consumes"
->
"configs.menus.Subscribe"

"pages.ConsumeCreate.ConsumerInformation"
->
"pages.ConsumeCreate.SubscriberInformation"

**Comment by kinfuy** [comment]
> 我们能不能让英文和中文一样，让他们更容易理解？
> 
> 同时“消费”到“订阅”和“消费者”到“订阅者”，如 “configs.menus.Consumes” -> “configs.menus.Subscribe”
> 
> "pages.ConsumeCreate.ConsumerInformation" -> "pages.ConsumeCreate.SubscriberInformation"

Modify only value? Need to modify the key at the same time?

File: inlong-dashboard/src/locales/en.json [hunk 5]
```diff
@@ -554,9 +554,9 @@
   "pages.ConsumeDashboard.status.NoStatus": "NoStatus",
   "pages.ConsumeDashboard.status.Passed": "Passed",
   "pages.ConsumeDashboard.status.Cancelled": "Cancelled",
-  "pages.ConsumeDashboard.NewConsume": "Create",
```
Comment (leezng) [inline inlong-dashboard/src/locales/en.json:557]: This key has been modified, but the page used has not been changed together

File: inlong-dashboard/src/i18n.ts [hunk 6]
```diff
@@ -27,7 +27,7 @@ const resources = {
     translation: {
       'configs.menus.Process': 'Approval',
       'configs.menus.Groups': 'Inlong Groups',
```
Comment (dockerzhang) [inline inlong-dashboard/src/i18n.ts:29]:
```suggestion
      'configs.menus.Groups': 'Integration',
```

File: inlong-dashboard/src/i18n.ts [hunk 7]
```diff
@@ -27,7 +27,7 @@ const resources = {
...
-      'configs.menus.Consumes': 'Inlong Consumes',
+      'configs.menus.Subscribe': 'Inlong Subscribe',
```
Comment (dockerzhang) [inline inlong-dashboard/src/i18n.ts:30]:
```suggestion
      'configs.menus.Subscribe': 'Subscription',
```

**Review by leezng — CHANGES_REQUESTED** [review]

File: inlong-dashboard/src/locales/en.json [hunk 8]
```diff
@@ -373,8 +373,8 @@
   "meta.Consume.FilterEnabled": "Filter Enabled",
   "meta.Consume.Yes": "Yes",
   "meta.Consume.No": "No",
-  "meta.Consume.Owner": "Consume Owners",
-  "meta.Consume.OwnersExtra": "Can view, modify consume info",
+  "meta.Consume.Owner": "Subscribe Owners",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/en.json:376]:
```suggestion
  "meta.Consume.Owner": "Subscription Owners",
```

File: inlong-dashboard/src/locales/en.json [hunk 9]
```diff
@@ -373,8 +373,8 @@
...
+  "meta.Consume.OwnersExtra": "Can view, modify subscribe info",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/en.json:377]:
```suggestion
  "meta.Consume.OwnersExtra": "Can view, modify subscription info",
```

File: inlong-dashboard/src/locales/en.json [hunk 10]
```diff
@@ -532,12 +532,12 @@
   "pages.Approvals.GroupId": "Inlong Group ID",
   "pages.Approvals.MyApplication": "My Application",
   "pages.Approvals.MyApproval": "My Approval",
-  "pages.ConsumeCreate.ConsumerInformation": "Consumer information",
+  "pages.ConsumeCreate.SubscriberInformation": "Subscriber information",
```
Comment (dockerzhang) [inline inlong-dashboard/src/locales/en.json:535]: Subscriber -> Subscription

**Review by healchow — APPROVED** [review]

**Comment by healchow** [comment]
@kinfuy Hi, I see you force-pushed a new commit. It is not recommended to use the force action, because it will overwrite the previous commit and review info.

**Review by leezng — APPROVED** [review]

**Comment by kinfuy** [comment]
> @kinfuy Hi, I see you force-pushed a new commit. It is not recommended to use the force action, because it will overwrite the previous commit and review info.

OK, thank you for your advice

**Pull request merged**